sandbox/handlers.js

```diff
@@ -62,7 +64,28 @@ async function hello(req, res, next) {
     next();
 }
 
+async function immunisation(req, res, next) {
+
+    write_log(res, "info", {
+        message: "immunisation",
+        req: {
+            path: req.path,
+            query: req.query,
+            headers: req.rawHeaders,
+            patientIdentifier: req.query["patient.identifier"].split("|")[1]
```

Comment:
`req.query["patient.identifier"].split("|")[1]` pull this out into its own variable and give it a name above